config.py

```diff
@@ -327,6 +334,11 @@ def load(cls):
         return configuration
 
     @classmethod
-    def _load(cls, fh):
-        lines = [x for x in fh if not x.strip().startswith("#")]
-        return json.loads("".join(lines))
+    def _load(cls, str):
+        lines = [x for x in str.split("\n") if not x.strip().startswith("#")]
+        config = json.loads("\n".join(lines))
+        if not config.get(cls.POLICIES):
+            config[cls.POLICIES] = {}
+        providers = config[cls.POLICIES].get(cls.ANALYTICS_POLICY, [])
```

Comment:
I'd like to see the default analytics policy (i.e. if you specify nothing in the config) to be the current default (i.e. analytics events get logged as CirculationEvent objects). This seems like the appropriate place to put this information.